exercises/space-age/.meta/generator/space_age_cases.rb

```diff
@@ -0,0 +1,31 @@
+class SpaceAgeCase < Generator::ExerciseCase
+  using Generator::Underscore
```

Comment:
Yep, it definitely is. I can swap my ```using``` statement in favor of requiring 'generator/exercise_case'```.